CODEOWNERS

```diff
@@ -0,0 +1,40 @@
+# This file is generated automatically by OpenSauced pizza-cli. DO NOT EDIT. Stay saucy!
+
+.github/ISSUE_TEMPLATE/bug_report.yaml @jpmcb
+.github/ISSUE_TEMPLATE/config.yaml @jpmcb
+.github/ISSUE_TEMPLATE/feature_request.yaml @jpmcb
+.github/workflows/auto-add-to-project.yml @jpmcb
+.github/workflows/release.yaml @jpmcb
+.github/workflows/test.yaml @jpmcb
+.golangci.yaml @jpmcb
+CHANGELOG.md @jpmcb @brandonroberts @nickytonline
+Dockerfile @jpmcb @nickytonline
+Makefile @jpmcb
+README.md @jpmcb
+cmd/auth/auth.go @jpmcb
+cmd/bake/bake.go @jpmcb
+cmd/bake/bake_test.go @jpmcb
+cmd/generate/codeowners/codeowners.go @jpmcb
+cmd/generate/codeowners/output.go @jpmcb @brandonroberts
+cmd/generate/codeowners/output_test.go @brandonroberts
+cmd/generate/codeowners/spec.go @jpmcb
+cmd/generate/codeowners/traversal.go @jpmcb
+cmd/generate/generate.go @jpmcb
+cmd/insights/contributors.go @jpmcb
+cmd/insights/insights.go @jpmcb
+cmd/insights/repositories.go @jpmcb
+cmd/insights/user-contributions.go @jpmcb
+cmd/insights/utils.go @jpmcb
+cmd/repo-query/repo-query.go @jpmcb
+cmd/root/root.go @jpmcb
+cmd/show/show.go @jpmcb
+cmd/version/version.go @jpmcb @nickytonline
+go.mod @jpmcb
+go.sum @jpmcb
+justfile @jpmcb @nickytonline
+npm/package.json @jpmcb @brandonroberts @nickytonline
```

Comment:
I didn't touch this file. Could this be from the CI run?

Reply:
hmmm, possibly. I didn't touch this file either. 🤔

Reply:
Must be from here

https://github.com/open-sauced/pizza-cli/commit/4de06e13677b569e77b5bcd489f3252cc80eff24

Reply:
Looks like the semantic-release action uses the author name / email when bumping the version:

![Screenshot 2024-08-28 at 4 25 54 PM](https://github.com/user-attachments/assets/6a2e8084-2a66-475a-b558-12ed2cbf8476)

Reply:
I guess just something to be aware of or maybe we omit commits coauthored with known bots for now?

Reply:
> I guess just something to be aware of or maybe we omit commits coauthored with known bots for now?

That'd be very hard to determine since the GitHub action is assuming the identity of the committer to commit to the `npm/package.json` repo which is why `Author: John McBride <john@opensauced.pizza>` is the author.

I'd say we punt on this for now and maybe note it in an issue: definitely an edge case.